src/Recoil.js

```diff
@@ -76,4 +80,5 @@ module.exports = {
 
   // Other functions
   isRecoilValue,
+  setBatch,
```

Comment:
The library user can call `setBatch` and pass a batching function exported by whatever React renderer they use. Due to the current setup (the lack of an `alternate-renderers` entrypoint), they'll still need `react-dom` as a dependency.